datafusion/core/src/physical_plan/sorts/sort.rs

```diff
@@ -1054,7 +1054,7 @@ mod tests {
     #[tokio::test]
     async fn test_sort_fetch_memory_calculation() -> Result<()> {
         // This test mirrors down the size from the example above.
-        let avg_batch_size = 5336;
+        let avg_batch_size = 6000;
```

Comment:
I suspect this relates to the recent changes that make arrays themselves slightly larger